integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -605,6 +622,9 @@ public int statfs(String path, Statvfs stbuf) {
   }
 
   private int statfsInternal(String path, Statvfs stbuf) {
+    if (mUfsEnabled) {
```

Comment:
does this mean statfs returns nothing?

Reply:
Yeah, this relies on our UnderFileSystem interface expose getCapacity related information but most object store does not have a upper limit
or we can return an arbitrary value e.g. a large enough capacity information